ethpm_types/contract_type.py

```diff
@@ -324,9 +324,34 @@ def get_deployment_bytecode(self) -> Optional[HexBytes]:
 
         return None
 
+    @cached_property
+    def identifier_definitions(self) -> Dict[str, str]:
```

Comment:
thought: Since we already have `method_identifiers` (which I think I copied from what Remix IDE was expecting when working on that), I am wondering if we should call this something like `definition_identifiers`?

Reply:
To keep in line with the rest of the terminology, would `selector_identifiers` be more appropriate?